config/configuration.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2019 Sonatype Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package config
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/sonatype-nexus-community/cheque/logger"
```

Comment:
Is this being used at all? Doesn't look like it. If you run `go build` before commits I THINK it strips unused imports (or tells you to).

Reply:
ok cool